crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -305,7 +307,14 @@ def disable_non_core_proposals
     end
 
     # and finally delete the roles itself
-    active_non_core_roles.each(&:destroy)
+    roles_to_disable.each do |role|
+      if role.name == "crowbar-config-default"
+        role.elements["crowbar-upgrade"] = []
+        role.save
+      else
+        role.destroy
```

Comment:
```crowbar-upgrade``` is an element role of the crowbar barclamp and part of the package. If we want to delete that we'd need a schema migration for that. But currently I don't think we want that.